device/delta/x86_64-delta_ag9032v1-r0/plugins/psuutil.py

```diff
@@ -45,9 +45,10 @@ def get_psu_presence(self, index):
             return False
         Base_bus_number = 39
         status = 0
+        self.psu_presence[2] += hex(index + Base_bus_number)
```

Comment:
>hex [](http://example.com/codeflow?start=32&length=3)

you added a "0x" compared with original behavior. Not sure if it works. To be safe, let's remove.